implementations/micrometer-registry-prometheus/src/main/java/io/micrometer/prometheusmetrics/PrometheusConfig.java

```diff
@@ -58,6 +60,19 @@ default Duration step() {
         return getDuration(this, "step").orElse(Duration.ofMinutes(1));
     }
 
+    /**
+     * @return an instance of {@link Properties} that contains Prometheus Java Client
+     * config entries, for example
+     * {@code io.prometheus.exporter.exemplarsOnAllMetricTypes=true}.
+     * @see https://prometheus.github.io/client_java/config/config/
+     */
+    @Nullable
+    default Properties prometheusProperties() {
+        Properties properties = new Properties();
+        properties.setProperty("io.prometheus.exporter.exemplarsOnAllMetricTypes", "true");
```

Comment:
We enable exemplars by default.
If somebody overrides this method, they can do this to reuse the Micrometer defaults but also define their own (see the tests):
```java
@Override
public Properties prometheusProperties() {
    Properties properties = new Properties();
    properties.putAll(PrometheusConfig.super.prometheusProperties());
    properties.setProperty("io.prometheus.exemplars.sampleIntervalMilliseconds", "42");
    return properties;
}
```